examples/aditof-demo/aditofdemoview.cpp

```diff
@@ -748,6 +748,7 @@ void AdiTofDemoView::render() {
                 }
             } else if (backspace) {
                 value = value.substr(0, value.size() - 1);
+                smallSignalThreshold = stoi(value);
```

Comment:
on the if branch, there is some validation to check if a value is not greater than (1 << 14) - 1.
What if you have a very large value and after pressing backspace the resulting value is still greater than that threshold? You're missing the validation here, on the else branch.

Reply:
The value from the box cannot be greater than (1 << 14) - 1. This is because of the UI which is really rudimental and there is not copy=paste capability, and also the cursor cannot be moved in the box (so you can only delete from the last digit). Also, the maximum number that can be introduced in the box is (1 << 14) - 1.

I though solved another bug: if all the digits are deleted with backspace, the application should treat this case as if no noise reduction is requested.